src/components/ListCard/ListCard.jsx

```diff
@@ -39,7 +40,13 @@ const ListCard = ({
   };
 
   return (
-    <Card id={id} title={title} size={size} onScroll={handleScroll} {...others}>
```

Comment:
Separate PR opened for this fix: https://github.com/carbon-design-system/carbon-addons-iot-react/pull/1692